src/docs/installation-systemd.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Use with systemd
+group: Installation
+priority: 3
+---
+
+You can optionally use `systemd` to manage photoview and start the program at boot.
+It also allows the program to run as its own system user, enhancing the security of the process.
+
+To get started, follow [Preparation](), [Download and build Photoview](), [Build the Web user-interface](), and [Build the API back-end]().
```

Comment:
Should this line be removed? It sounds a lot like the one below :)

```suggestion
```

Reply:
Oh, ha, yeah. I re-wrote the line, then forgot to remove the original version. Good catch!

Reply:
I though that was the case, anyways thank you for helping out!